catalog/gitops/README.md

```diff
@@ -1,6 +1,9 @@
-# CSR Git Ops Pipeline
+# GitOps package
+
+GitOps CI/CD pipeline for use with ACM.
+
+After installing this blueprint, you will be able to:
 
-This blueprint instantiates a GitOps CI/CD pipeline using Cloud Source Repositories and Cloud Build. After installing this blueprint, you will be able to:
```

Comment:
Why was this removed? It's important that CSR usage is stated up front.

Reply:
Removed on accident while trying to be one concise. I'll add it back. I didn't mean to hide the detail that it uses CSR from the description.